certora/specs/AccessControlDefaultAdminRules.spec

```diff
@@ -484,3 +432,50 @@ rule rollbackDefaultAdminDelay(env e) {
   assert success => pendingDelaySchedule_(e) == 0,
     "Pending default admin delay is reset";
 }
+
+/* 
+┌─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┐
+│ Rule: pending default admin and the delay can only change along with their corresponding schedules                  │
+└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+*/
+rule pendingValueAndScheduleCoupling(env e, address newAdmin, uint48 newDelay) {
```

Comment:
Personally I don't really understand why this rule verifies a property for both default admin and delay, which are separate values. IMO, it would be more correct to verify them separately so that the first one doesn't constrain the state on which the second one is verified.

That said, I don't consider this a blocker.

Reply:
I see your point. We [agreed on merging](https://github.com/OpenZeppelin/openzeppelin-contracts/pull/4180#discussion_r1178014239) them to have fewer rules, but the state constraints kinda worry me.

I also don't consider it a blocker